PYTHIA8/AliPythia8/AliDecayerPythia8.h

```diff
@@ -39,15 +39,15 @@ class AliDecayerPythia8 : public TVirtualMCDecayer {
   AliDecayerPythia8(const AliDecayerPythia8&);
   AliDecayerPythia8 operator=(const AliDecayerPythia8&);
   void     SwitchOffHeavyFlavour();
-  void     ForceHadronicD(Int_t optUser4Bodies = 1,Int_t optUseDtoV0 =1, Int_t optForceLcChannel = 0, Int_t optForceXicChannel = 0, Int_t optForceDsChannel = 0);
+  void     ForceHadronicD(Int_t optUser4Bodies = 1,Int_t optUseDtoV0 =1, Int_t optForceLcChannel = 0, Int_t optForceXicChannel = 0, Int_t optForceDsChannel = 0, Int_t optForceOmegacChannel = 0);
   void  ForceBeautyUpgrade();
   AliTPythia8*  fPythia8;          // Pointer to pythia8
   Int_t         fDebug;            // Debug level
  
   Decay_t       fDecay;            //  Forced decay mode
   Bool_t        fHeavyFlavour;     //! Flag for heavy flavors
   static Bool_t fgInit;            //! initialization flag 
-  ClassDef(AliDecayerPythia8, 2) // Particle Decayer using Pythia8
+  ClassDef(AliDecayerPythia8, 3) // Particle Decayer using Pythia8
```

Comment:
Why did you increase the ClassDef? You did not add any data member, unless I am overlooking something.

Reply:
You are right, I did not add new data member, just modified something.  Sorry, I do not know the rules in detail.  I will keep '2', and recommit this file again soon. Thanks a lot.

Reply:
@chiarazampolli Could you please have a look about the new PRs and approve it. Thank you very much